kinder/pkg/actions/waiter.go

```diff
@@ -0,0 +1,273 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package actions
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+
+	kcluster "k8s.io/kubeadm/kinder/pkg/cluster"
+	"sigs.k8s.io/kind/pkg/exec"
+)
+
+// waitNewControlPlaneNodeReady waits for a new control plane node reaching the target state after init/join
+func waitNewControlPlaneNodeReady(kctx *kcluster.KContext, kn *kcluster.KNode, flags kcluster.ActionFlags) error {
+	fmt.Printf("==> Waiting for Node and control-plane pods getting Ready (timeout %s) 📦\n", flags.Wait)
```

Comment:
minor change:
`fmt.Printf("==> Waiting for Node and control-plane Pods to become Ready (timeout %s) 📦\n", flags.Wait)`